airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/config.py

```diff
@@ -143,6 +143,14 @@ class TestConfig(BaseConfig):
 
 
 class Config(BaseConfig):
+    class TestMode(str, Enum):
+        strict = "strict"
+        medium = "medium"
+        light = "light"
```

Comment:
Not sure it's worth defining these now since we don't know what they mean. Like, `test_mode=strict` makes sense to me that it would... be more strict. But for e.g. `test_mode=medium` it's not really apparent what that's going to do, specially because there's no mention of strictness level there.